sites/all/themes/wille/template.php

```diff
@@ -112,22 +112,23 @@ function wille_ting_collection_view_alter(&$build) {
  * Wraps the cover in a link to the material.
  */
 function wille_ting_object_cover($variables) {
-  $ding_entity_id = $variables['elements']['#object']->ding_entity_id;
-  $type = $variables['meta_for_labels']['type'];
-  $quota_explanation = t('This material is a !type and is not on your quota', ['!type' => $type]);
-  if (isset($variables['meta_for_labels']['on_quota']) && $variables['meta_for_labels']['on_quota']) {
-    $quota_explanation = t('This material is a !type and is on your quota', ['!type' => $type]);
-  }
-  $attributes['aria-label'] = $quota_explanation;
   $attributes = array(
     'class' => implode(' ', $variables['classes']),
   );
+  if (isset($variables['label_info']) && isset($variables['label_info']['type']) && isset($variables['label_info']['on_quota'])) {
```

Comment:
May be written as `isset($variables['label_info']['type'], $variables['label_info']['on_quota'])` which may be easier to read (cf. https://www.php.net/manual/en/function.isset.php).